src/lavinmq/config.cr

```diff
@@ -60,6 +60,8 @@ module LavinMQ
     property consumer_timeout : UInt64? = nil
     property consumer_timeout_loop_interval = 60 # seconds
     property default_consumer_prefetch = UInt16::MAX
+    property yield_each_received_bytes = 131_072    # after how many received bytes the read_loop fiber yields
```

Comment:
Yeah, need to figure out how to explain it in a way that's easy to understand though... I'll give it some more thought! 
Maybe something along the lines of "a higher value generally increases throughput on a single consumer, but could cause issues if consuming slowly"

Reply:
Updated to: 
```
    property yield_each_received_bytes = 131_072 # number of bytes received from a publisher before yielding to other fibers
    property yield_each_delivered_bytes = 1_048_576 # number of bytes sent to a consumer before yielding to other fibers
```
is that better? 

It's hard to describe it much better in a short sentence IMO, and I don't want to add a long description here. I think it would be better if we had some more thorough documentation of "advanced" config vars in the docs!